packages/evm/src/precompiles/09-blake2f.ts

```diff
@@ -1,7 +1,7 @@
 import { bytesToHex } from '@ethereumjs/util'
 
+import { EVMError, EvmError } from '../errors.ts'
```

Comment:
What's this dual-naming about? This seems confusing

Reply:
Fixed.